en/assembly/quick_start_cuav_v5_nano.md

```diff
@@ -49,12 +49,13 @@ Connect to the flight control GPS interface using a cable.
 ## Safety Switch
 
 The dedicated safety switch that comes with the V5+ is only required if you are not using the recommended Neo GPS (which has an inbuilt safety switch).
-If you are flying without the GPS you must attach the switch directly to the  GPS1  port in order to be able to arm the vehicle and fly (If you use the old 6-pin GPS, please read the definition of the bottom interface to change the line.).
+
+If you are flying without the GPS you must attach the switch directly to the GPS1 port in order to be able to arm the vehicle and fly (If you use the old 6-pin GPS, please read the definition of the bottom interface to change the line).
 
 
 ## Buzzer
 
-If you do not use the recommended GPS, there may be problems with the buzzer not working.
+If you do not use the recommended GPS the buzzer may not work.
```

Comment:
Is there any information on how/where you should attach an external buzzer if not using the recommended GPS?

Reply:
This can be a bit difficult.

Reply:
Yes. So what do you advise? 

It would be better to point them to guidance on how to do this, but if there is none, how about we just remove the text "You may need to add an external buzzer instead."?